content/en/integrations/content_security_policy_logs.md

```diff
@@ -182,6 +182,14 @@ If you are using Session Replay, make sure to allow workers with `blob:` URI sch
 worker-src blob:;
 ```
 
+Alternatively, starting from [version TODO][8], you can self-host the Datadog Browser SDK Worker JavaScript file and provide the `workerUrl` option to initialize the [RUM Browser SDK][8]. To do so, you can either:
+
+* download it from https://unpkg.com/@datadog/browser-worker, and store it alongside your Web application assets;
+
+* or install the [`@datadog/browser-worker` NPM package][9] and use your build tool to include it in the built assets (see documentation for [Webpack 4][10], [Webpack 5][11], [Vite][12], [Rollup][13]).
+
+Keep in mind that you need to host it on the *same origin* as your Web application. It cannot be hosted on a separate domain (ex: no third party CDN host) or a different scheme. This is a browser restriction (see [MDN documentation][14]: "This script must obey the same-origin policy").
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we should maybe add a note saying something about the worker version matching the sdk version

Reply:
Done. I mentioned "*major* version", as CDN users won't maintain an exactly matching version and I wanted to keep it simple, but I'm open to suggestions